### DISCUSSION TITLE
B站直播间 聊天页面滚动条无法滑动

### DISCUSSION THREAD

**Discussion by warmy1** [F9y4ng/GreasyFork-Scripts#143] [问答专区(Question / Answer)] [answered]
如题
关闭脚本后正常

**Comment by F9y4ng** [comment, marked as answer]
将实验性功能**字体比例缩放**关闭即可（设置为1.000，保存为站点独享）。

## 问题复现
bilibili直播间聊天区域滚动代码并未对css:zoom的情况进行兼容，造成高度获取不准确，从而无法正常滚动。

## 建议操作
关闭**字体比例缩放**功能，有缩放必要请使用浏览器缩放替代(ctrl++,ctrl+-)。

**Reply by warmy1** [reply]
好的，谢谢解答！
